pokemon/src/main/java/gamarket/EncounterGUI.java

```diff
@@ -1,125 +1,1075 @@
+                + EncounterGUI.setColor(1) + "-fx-font-size: 32px;" + "-fx-text-align: center;"
+                + "-fx-text-base-color: rgb(0,0,0);");
+        }});
+        move2.addEventHandler(MouseEvent.MOUSE_EXITED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent e) {
+                move2.setStyle("-fx-background-radius: 400px;" + "-fx-pref-width: 400px; " + "-fx-border: none; "
+                + "-fx-border-radius: 400px;" + "-fx-border-width: 7px;"  + "-fx-border-color: slategray;"
+                + EncounterGUI.setColor(1) + "-fx-font-size: 32px;" + "-fx-text-align: center;" 
+                + "-fx-text-base-color: rgb(0,0,0);");
+        }});
+        move3.addEventHandler(MouseEvent.MOUSE_ENTERED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent e) {
+                move3.setStyle("-fx-background-radius: 400px;" + "-fx-pref-width: 400px; " + "-fx-border: none; "
+                + EncounterGUI.setColor(2) + "-fx-border-color: rgb(0,0,0);"
+                + "-fx-border-radius: 400px;" + "-fx-border-width: 7px;" + "-fx-font-size: 32px;"
+                + "-fx-text-align: center;" + "-fx-text-base-color: rgb(0,0,0);");
+        }});
+        move3.addEventHandler(MouseEvent.MOUSE_EXITED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent e) {
+                move3.setStyle("-fx-background-radius: 400px;" + "-fx-pref-width: 400px; " + "-fx-border: none; "
+                + EncounterGUI.setColor(2)  + "-fx-border-color: slategray;"
+                + "-fx-border-radius: 400px;" + "-fx-border-width: 7px;" + "-fx-font-size: 32px;"
+                + "-fx-text-align: center;" + "-fx-text-base-color: rgb(0,0,0);");
+        }});
+        move4.addEventHandler(MouseEvent.MOUSE_ENTERED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent e) {
+                move4.setStyle("-fx-background-radius: 400px;" + "-fx-pref-width: 400px; " + "-fx-border: none; "
+                + "-fx-border-color: rgb(0,0,0);" + "-fx-border-radius: 400px;" + "-fx-border-width: 7px;"
+                + EncounterGUI.setColor(3) + "-fx-font-size: 32px;" + "-fx-text-align: center;"
+                + "-fx-text-base-color: rgb(0,0,0);");
+        }});
+        move4.addEventHandler(MouseEvent.MOUSE_EXITED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent e) {
+                move4.setStyle("-fx-background-radius: 400px;" + "-fx-pref-width: 400px; " + "-fx-border: none; "
+                + "-fx-border-radius: 400px;" + "-fx-border-width: 7px;"  + "-fx-border-color: slategray;"
+                + EncounterGUI.setColor(3) + "-fx-font-size: 32px;" + "-fx-text-align: center;"
+                + "-fx-text-base-color: rgb(0,0,0);");
+        }}); 

```

Comment:
alot of these mouse exit and mouse enter styles look to be the same. If so use the same variable.